LayoutTests/media/media-source/media-managedmse-airplay.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html> <!-- webkit-test-runner [ ManagedMediaSourceEnabled=true MediaSourceEnabled=true ] -->
+<html>
+<head>
+    <title>managedmediasource-airplay</title>
+    <script src="../../media/media-source/media-source-loader.js"></script>
+    <script src="../../media/video-test.js"></script>
+    <script src="../../media/utilities.js"></script>
+    <script>
+    var loader;
+    var source;
+    var sourceBuffer;
+    var sourceElement;
+    var sourceOpenAllowed = false;
+
+    function loaderPromise(loader) {
+        return new Promise((resolve, reject) => {
+            loader.onload = resolve;
+            loader.onerror = reject;
+        });
+    }
+
+    window.addEventListener('load', async event => {
+        try {
+            findMediaElement();
+
+            let manifests = [ 'content/test-opus-manifest.json', 'content/test-vorbis-manifest.json', 'content/test-48khz-manifest.json', 'content/test-xhe-aac-manifest.json' ];
+            for (const manifest of manifests) {
+                loader = new MediaSourceLoader(manifest);
+                await loaderPromise(loader);
+                if (ManagedMediaSource.isTypeSupported(loader.type()))
+                    break;
+            }
+
+            waitFor(video, 'error').then(failTest);
+
+            run('source = new ManagedMediaSource()');
+            run('video.src = URL.createObjectURL(source)');
+
+            waitFor(source, 'sourceopen').then(() => {
+                if (!sourceOpenAllowed)
+                    failTest()
+            });
+
+            await sleepFor(100);
+            run('video.setAttribute("x-webkit-airplay", "deny")');
```

Comment:
I would use the standard [disableRemotePlayback](https://w3c.github.io/remote-playback/#the-disableremoteplayback-attribute) attribute instead of `x-webkit-airplay` in the tests and in your commit message.

Reply:
I've added the ability to do either, and will use the Remote Playback API in the test.